pkgs/by-name/br/breseq/tests/breseq.nix

```diff
@@ -0,0 +1,25 @@
+{
+  runCommand,
+  breseq,
+}:
+
+let
+  inherit (breseq) pname version;
+in
+
+runCommand "breseq-tests" { meta.timeout = 60; } ''
+  echo "Testing breseq - breseq executable"
+  if [[ `${breseq}/bin/breseq --version` != *"${version}"*  ]]; then
```

Comment:
Take a look at `versionCheckHook` to do the check directly via the derivation declaration